### PR TITLE
Add supply chain support for the yieldlab adapter

### PR DESCRIPTION
**Pull request by Mufas61**

### CODE REVIEW

**Comment by Mufas61** [comment]
Hey guys, we were wondering if it would somehow be possible to add our own colleagues as required reviewers?

**Comment by bsardo** [comment]
Hi @Mufas61, what are you trying to do that you're not already able to do on the PR? Anyone should be able to comment on the PR but you would still need two approvals from the PBS-Go team to merge.

**Comment by Mufas61** [comment]
> Hi @Mufas61, what are you trying to do that you're not already able to do on the PR? Anyone should be able to comment on the PR but you would still need two approvals from the PBS-Go team to merge.

Hi @bsardo, in addition to your review, we would like to have a approval from one of our team as a requirement for the merge. Preferably as a step before your review. Of course we are able to comment on the PR, but sometimes you guys are faster than our team and the PRs are already approved before any of us had a look.
If you don't have a workflow for this, it shouldn't be a problem. In that case, we would just move the step to our fork.

File: adapters/yieldlab/yieldlab_test.go
Comment (hhhjort) [inline adapters/yieldlab/yieldlab_test.go:162]: You have the same test listed twice?

**Comment by hhhjort** [comment]
Looking good, just one comment about a redundant test case. I won't approve until I see an approval comment from the yieldlab team to make sure we don't jump the gun.

As far as I know github doesn't have the feature to add a one-off reviewer to a PR, but if you know of a way, please let us know.

**Comment by bsardo** [comment]
> > Hi @Mufas61, what are you trying to do that you're not already able to do on the PR? Anyone should be able to comment on the PR but you would still need two approvals from the PBS-Go team to merge.
> 
> Hi @bsardo, in addition to your review, we would like to have a approval from one of our team as a requirement for the merge. Preferably as a step before your review. Of course we are able to comment on the PR, but sometimes you guys are faster than our team and the PRs are already approved before any of us had a look. If you don't have a workflow for this, it shouldn't be a problem. In that case, we would just move the step to our fork.

Hi @Mufas61, unfortunately we aren't able to add people who aren't prebid members as reviewers. One solution might be to have your PR author ping internal team members in a comment when they open a new PR. Once they think everything looks good, they can ping us and we'll begin our formal review.

**Comment by Mufas61** [comment]
@kippsterr test

**Comment by Mufas61** [comment]
> Looking good, just one comment about a redundant test case. I won't approve until I see an approval comment from the yieldlab team to make sure we don't jump the gun.
> 
> As far as I know github doesn't have the feature to add a one-off reviewer to a PR, but if you know of a way, please let us know.

File: adapters/yieldlab/yieldlab_test.go
Comment (Mufas61) [reply adapters/yieldlab/yieldlab_test.go]: removed the duplicate.

**Comment by bsardo** [comment]
@Mufas61, I saw you closed this and then re-opened, which I assume was by mistake. Just checking to see if you're ready for us to review. If not, ping us when you're ready.

**Comment by Mufas61** [comment]
> @Mufas61, I saw you closed this and then re-opened, which I assume was by mistake. Just checking to see if you're ready for us to review. If not, ping us when you're ready.

Hi @bsardo, yes the closing was just a misclick. We will ping you as soon as one of our team has completed the review.

**Comment by bsardo** [comment]
@Mufas61, I see an approval from @brushmate. Would you like us to give this a formal review now?

**Comment by kippsterr** [comment]
> @Mufas61, I see an approval from @brushmate. Would you like us to give this a formal review now?

Yes @bsardo, please go for it :)

**Review by hhhjort — APPROVED** [review]
LGTM

File: adapters/yieldlab/yieldlab.go
Comment (AlexBVolcy) [inline adapters/yieldlab/yieldlab.go:360]: I'm going to confirm with the team that this ignoring of error handling is okay. But in most circumstances, we usually try to avoid non-error handling.

**Review by AlexBVolcy — COMMENTED** [review]
Looks good, just going to confirm the non-error handling is okay or not.

File: adapters/yieldlab/yieldlab.go
Comment (Mufas61) [reply adapters/yieldlab/yieldlab.go:360]: req.Source.Ext is a freeform field, so we can not be sure that customers will use it to pass the supply chain parameter. This will change with OpenRTB 2.6. There is a specified field for it.

**Review by AlexBVolcy — APPROVED** [review]
The ignoring of that unmarshal error is ok! And everything else LGTM

**Pull request merged**
